### PR TITLE
TINY-10128: For streamContent: true iframes, use numerical value for scrolling and only scroll after iframe load

### PR DESCRIPTION
**Pull request by mkzhx**
Related Ticket: TINY-10128

Description of Changes:
* Use a numerical value for Window.scrollTo instead of body.scrollHeight (in case body is `null`)
* On all browsers, wait for `load` before attempting to scroll, as trying to scroll before iframe is loaded does not work

Pre-checks:
* [x] Changelog entry added
* [x] ~Tests have been added (if applicable)~ - already have tests for interval updates, but this fix targets a rare edge case
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] ~Docs ticket created (if applicable)~

GitHub issues (if applicable):

### CODE REVIEW

**Review by TheSpyder — APPROVED** [review]
Does this really do anything if `body` is null? Seems strange that it would be visible but not set in the properties.

I wonder if we should just always use the large number and not the scroll height 😂

**Review by shanmen-tiny — APPROVED** [review]

**Comment by mkzhx** [comment]
> Does this really do anything if `body` is null? Seems strange that it would be visible but not set in the properties.
> 
> I wonder if we should just always use the large number and not the scroll height 😂

@TheSpyder Yeah I agree, let's just always use the fallback value since it is not feasible to know when the body is `null` anyway. I've also refined the fallback value to 2147483647 (2^31 - 1) since I found that Safari can take up to 32 bits in that parameter.

File: modules/tinymce/CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Contents would not be removed from the drag start source if dragging and dropping internally into a transparent block element. #TINY-9960
 - In some cases pressing enter would scroll the entire page. #TINY-9828
 - The border styles of a table were incorrectly split into a longhand form after table dialog updates. #TINY-9843
+- An "Uncaught TypeError: Cannot read properties of null" error would sometimes be thrown when updating the content of a `streamContent: true` iframe dialog component. #TINY-10128
```
[the remark this answers is not in the document]
Comment (mkzhx) [reply modules/tinymce/CHANGELOG.md:35]: The error also shows up in the UI as per our design. I think it would be best to be specific on the type of error that shows up in case people actively search for that in the changelog.

**Review by shanmen-tiny — APPROVED** [review]

File: modules/tinymce/CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheSpyder) [reply modules/tinymce/CHANGELOG.md:35]: Only if the dialog catches the error; TinyMCE won't show the error visually itself.

Remember the boundary between what we do, and what integrators can do in their own code.

**Review by TheSpyder — APPROVED** [review]
I'm glad we got rid of the Safari code. I understand the reason it was there, but the code was always pushing a bit too close to the line on the other browsers.

**Review by ltrouton — APPROVED** [review]
